docs/api/rebac/groups.mdx

```diff
@@ -0,0 +1,28 @@
+---
+sidebar_position: 7
+title: Groups API
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+Groups API lets you manage your objects in a group. You can create, update, delete, and list groups.
+Each group will have roles associated with it, and every user in the group will inherit the roles of the group.
```

Comment:
maybe something on the lines of "You can use groups to organize users and assign roles collectively by assigning roles to the group. Each group can have roles associated with it, and users in the group can inherit these roles from the resource it is linked to."